### PR TITLE
Add more meetup events.

### PR DESCRIPTION
**Pull request by josisusan**
Task done:

- Add more meetup events from 2014 and 2015.
- Re-organize meetup events according to descending date (current being the latest).
- Fix typo in company name.

### CODE REVIEW

File: js/meetup/data.js [hunk 1]
```diff
@@ -89,10 +89,10 @@ const meetups = [
         title: 'History of RubyNepal',
         description: null,
         speaker: 'Saroj Maharjan',
-        twitter_username: 'zoraslapen',
+        twitter_username: 'zorasplan',
```
Comment (zoras) [inline js/meetup/data.js]: 🙅‍♂

Comment (josisusan) [reply js/meetup/data.js]: lol

Comment (josisusan) [reply js/meetup/data.js]: fixed

File: _data/meetup.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
-- episode: 9
+- episode: 12
   title: RoR Himalayan Meetup at Kathmandu
-  date: 2018/05/13
-  start_time: '11:00 am'
-  end_time: '2:00 am'
+  date: '2018-05-13'
+  start_time: 11:00 am
+  end_time: 2:00 am
```
Comment (zoras) [inline _data/meetup.yml]: It should be 2 pm everywhere

**Comment by josisusan** [comment]
Merging this to master.

**Pull request merged**